modules/apps/transfer/types/packet.go

```diff
@@ -23,6 +23,11 @@ var (
 	_ ibcexported.PacketDataProvider = (*FungibleTokenPacketDataV2)(nil)
 )
 
+const (
+	JsonEncoding  = "application/json"
+	ProtoEncoding = "application/x-protobuf"
```

Comment:
> Would these be better named as `EncodingJSON` and `EncodingProto(buf)`?

Agreed

> The MIME format is exactly what is expected, right? i.e. application/json rather than "json" as it was previously?

Based on the spec that is my understanding, yes